k8s/kustomize/helm-servers/patch_mlserver.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: mlops.seldon.io/v1alpha1
 kind: Server
 metadata:
   name: mlserver
+  labels:
```

Comment:
is `labels: HACK_REMOVE_ME{{ toJson .Values.mlserver.labels }}` not possible?

Reply:
I don't think so - a valid label would be something like `HACK_REMOVE_ME: HACK_REMOVE_ME{{ toJson .Values.mlserver.labels }}`.

Reply:
This might work for a `serverconfig.component` though...

Reply:
Got it now, kustomize is a bit smarter regarding labels than we would have liked in this particular case.

For the ugliest hack immaginable,
`labelz: HACK_REMOVE_ME{{....}}` would work under matadata, with us processing `labelz` via sed.....